pkg/odo/cli/build_images/build_images.go

```diff
@@ -79,7 +79,7 @@ func NewCmdBuildImages(name, fullName string) *cobra.Command {
 	}
 
 	// Add a defined annotation in order to appear in the help menu
-	buildImagesCmd.Annotations = map[string]string{"command": "utility"}
+	buildImagesCmd.Annotations = map[string]string{"command": "main"}
```

Comment:
Do we really want to put this up in the "main" section? I remember we added it more as a utility.

Reply:
yup, as per discussions with @kadel